src/test/scala/pick16SCandidates.scala

```diff
@@ -106,13 +106,21 @@ case object pick16SCandidates extends FilterData(
       if (commonID != fasta.getV(header).id)
         sys.error(s"ID [${commonID}] is not found in the FASTA. Check RNACentral filtering.")
 
+      /* This predicate determines whether the *sequence* value is OK, and will be kept. */
+      val sequenceIsOK: String => Boolean =
+        seq: String =>
+          ( seq.length >= minimum16SLength )              &&
+          ( (seq.count(_ == 'N') / seq.length) <= 0.01 )  &&
+          ( !(seq containsSlice "NNNNNNNN") )
```

Comment:
Previous condition is self-explanatory: you want to have less than some percentage of `N`s. This one looks a bit strange because when you read it you need to count those `N`s and this minimal number looks quite arbitrary. I suggest:

```scala
val Ns = seq.count(_ == 'N')

seq.length >= minimum16SLength &&
(Ns / seq.length) <= 0.01 &&
Ns < 8
```